decoders/0005-wazuh_decoders.xml

```diff
@@ -6,7 +6,51 @@
 -->
 
 <decoder name="wazuh">
-  <prematch>^wazuh: </prematch>
+  <prematch>^wazuh: |^ossec: </prematch>
+  <type>wazuh</type>
+</decoder>
+
+<decoder name="wazuh-logcollector">
+  <type>wazuh</type>
+  <prematch>^\d\d\d\d/\d\d/\d\d \d\d:\d\d:\d\d ossec-logcollector|^\d\d\d\d/\d\d/\d\d \d\d:\d\d:\d\d wazuh-logcollector</prematch>
+  <regex offset="after_prematch">^\(\d+\): (\.)</regex>
+  <order>extra_data</order>
+</decoder>
+
+<decoder name="wazuh-syscheck-audit">
+  <parent>wazuh</parent>
+  <type>wazuh</type>
+  <prematch offset="after_parent">^Audit:</prematch>
+  <regex offset="after_prematch">^ (\.+)</regex>
+  <order>extra_data</order>
+</decoder>
+
+<decoder name="wazuh-agent">
+  <parent>wazuh</parent>
+  <type>wazuh</type>
+  <prematch offset="after_parent">^Agent started:</prematch>
+  <regex offset="after_prematch">^ '(\S+\S)'</regex>
+  <order>extra_data</order>
+  <fts>name, location, extra_data</fts>
+</decoder>
+
+<decoder name="wazuh-alert1">
+   <parent>wazuh</parent>
+   <prematch>^ossec: Alert Level:|^wazuh: Alert Level:</prematch>
```

Comment:
This log is only created in `read_ossecalert()` and `OS_Alert_SendSyslog()`
I think these two methods are only part of Wazuh Manager, so it shouldn't be necessary to keep both versions of the decoder.

Reply:
+1

Reply:
You are right, those logs aren't present in the agents